client/src/components/LinkToPreviewed.js

```diff
@@ -0,0 +1,121 @@
+import { PopoverInteractionKind } from "@blueprintjs/core"
+import AvatarDisplayComponent from "components/AvatarDisplayComponent"
+import { OBJECT_TYPE_TO_MODEL } from "components/Model"
+import ModelPreview from "components/ModelPreview"
+import ModelTooltip from "components/ModelTooltip"
+import _isEmpty from "lodash/isEmpty"
+import * as Models from "models"
+import PropTypes from "prop-types"
+import React from "react"
+import { Link } from "react-router-dom"
+
+const LinkToPreviewed = ({
+  componentClass,
+  children,
+  showIcon,
+  showAvatar,
+  whenUnspecified,
+  className,
+  modelType,
+  model,
+  style,
+  previewId,
+  ...componentProps
+}) => {
+  componentProps.className = className
```

Comment:
I would not assign this here, but add `className={className}` to the props of `<LinkToComponent />` below.